frontend/utils/wrapWithIdentifier.ts

```diff
@@ -0,0 +1,19 @@
+import type { ApiPagingResponse } from '~/types/api/common'
+/**
+ * This function wraps each element in an `ApiPagingResponse` interface with a unique id field `wrapped_identifier`.
+ * @param response The api response to wrap
+ * @param getIdFromRow A function that takes a row and builds a unique identifier
+ * @returns The wrapped response
+ */
+export function wrapWithIdentifier<T>(response: ApiPagingResponse<T> | undefined, ...keys: (keyof T)[]) {
+  if (!response) {
+    return
+  }
+  return {
+    data: response.data.map(row => ({
+      ...row,
+      wrapped_identifier: keys.map(key => row[key]).join('-'),
+    })),
+    paging: response.paging,
+  }
+}
```

Comment:
maybe think about generalize this function (is it important for the function that there has to be a row?)

Reply:
Pushed the changes, ptal.
As for the generalization, since this func takes `ApiPagingResponse`s, which only concern tables, the function can expect there to be rows. If we want to add identifiers so something other than `ApiPagingResponse`s, we can think about generalizing more.